src/commands/e2e/ConfigTemplate.ts

```diff
@@ -108,7 +109,7 @@ exports.config = {
             project: '${e2eFolder}/tsconfig.json'
         });
         return new Promise(function(resolve) {
-            return request('${baseUrl}${context}${this.listPluginsURL}?testSetupHandlerE2EToken=${e2eToken}', function(error, response, body) {
+            return request('${baseUrl}${context.context}${context.tenantId}${this.listPluginsURL}?testSetupHandlerE2EToken=${e2eToken}', function(error, response, body) {
```

Comment:
Die Logik hier muss letztendlich nur angepasst werden, dass die Slashes hier richtig gejoint werden - in der `E2E.ts` sollte es vermutlich keine Änderungen geben... Auch aufpassen: sowohl `context` als auch `tenantId` können leer sein (beides vollkommen valide - z.B. auf einem Single Tenant System ohne Context wie unsere Base 😉 )